cmd/acceptance/main.go

```diff
@@ -142,21 +173,19 @@ func main() {
 	}
 }
 
-var _ = Describe("Acceptance", func() {
```

Comment:
This diff is horrible, better to just view the file. I've inlined the describe block into the run command, no biggie.